matrix/commands.py

```diff
@@ -1321,6 +1342,64 @@ def predicate(event_id, line):
     return W.WEECHAT_RC_OK
 
 
+@utf8_decode
+def matrix_reply_command_cb(data, buffer, args):
+    def predicate(event_id, line):
+        event_tag = SCRIPT_NAME + "_id_{}".format(event_id)
+        tags = line.tags
+
+        if event_tag in tags:
+            return True
+        return False
+
+    for server in SERVERS.values():
+        if buffer in server.buffers.values():
+            room_buffer = server.find_room_from_ptr(buffer)
+
+            # event_id, reply = parse_reply_args(args)
+            event_id, reply = parse_redact_args(args)
+
+            if not event_id:
+                message = (
+                    "{prefix}matrix: Invalid command "
+                    "arguments (see /help reply)"
+                ).format(prefix=W.prefix("error"))
+                W.prnt("", message)
+                return W.WEECHAT_RC_ERROR
+
+            lines = room_buffer.weechat_buffer.find_lines(
+                partial(predicate, event_id), max_lines=1
+            )
+
+            if not lines:
+                room_buffer.error(
+                    "No such message with event id "
+                    "{event_id} found.".format(event_id=event_id))
+                return W.WEECHAT_RC_OK
+
+            formatted_data = Formatted.from_input_line(reply)
+            server.room_send_message(
+                room_buffer,
+                formatted_data,
+                "m.text",
+                ignore_unverified_devices=True,
```

Comment:
I don't think we should do that considering the behavior of the script in other places.

Reply:
OK, line removed.